src/stores/viewStore.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import { defineStore } from 'pinia'
+import { Filter } from 'rdf-cube-view-query'
+import { shallowRef } from 'vue'
+
+const useViewStore = defineStore('viewStore', () => {
```

Comment:
It provides an accessor for the current view that's in memory.
Also, helpers to add and remove filters from a dimension (column)